maddy.conf.5.scd

```diff
@@ -539,6 +539,45 @@ Refuse to accept messages larger than `value` bytes. The default is 32 MiB.
 
 Enable verbose logging only for this configuration block.
 
+# QUEUE MODULE
+
+Queue module buffers messages on disk and retries delivery multiple times to
+ensure reliable delivery.
+
+```
+queue {
+    workers 16
+    max_tries 8
+    target dummy
+}
+```
+
+Valid configuration directives:
+
+## location
+
+File system directory to use to store queued messages.
+
+Default is /state_directory/configuration_block_name.
```

Comment:
`$MADDYSTATE/<configuration_block_name>` would be better IMHO

Reply:
ce1b726